firestore/integration_test_internal/src/transaction_test.cc

```diff
@@ -743,21 +745,24 @@ TEST_F(TransactionTest, TestMaxAttempts) {
 
   Future<void> run_transaction_future = firestore->RunTransaction(
       options,
-      [update_count, &doc](Transaction& transaction, std::string&) -> Error {
-        SCOPED_TRACE("Update callback; update_count=" +
-                     std::to_string(update_count->load()));
+      [update_count, &doc](Transaction& transaction,
+                           std::string& error_message) -> Error {
+        SCOPED_TRACE(absl::StrCat("Update callback; update_count=",
```

Comment:
This change to absl::StrCat() is merely a code readability improvement IMO and does not actually improve any error reporting. Its change here is merely out of convenience.

Reply:
Nevermind; I've reverted this change since using absl::StrCat opened up a whole can of worms regarding linking absl. See https://github.com/firebase/firebase-cpp-sdk/pull/1015.